crates/testing/src/helpers.rs

```diff
@@ -376,18 +367,16 @@ pub async fn build_da_certificate<TYPES: NodeType, V: Versions>(
 ) -> DaCertificate<TYPES> {
     let encoded_transactions = TestTransaction::encode(&transactions);
 
-    let da_payload_commitment = vid_commitment(
-        &encoded_transactions,
-        quorum_membership.total_nodes(epoch_number),
-    );
+    let da_payload_commitment =
+        vid_commitment(&encoded_transactions, membership.total_nodes(epoch_number));
 
     let da_data = DaData {
         payload_commit: da_payload_commitment,
     };
 
     build_cert::<TYPES, V, DaData, DaVote<TYPES>, DaCertificate<TYPES>>(
         da_data,
-        da_membership,
+        membership,
```

Comment:
this should pass the da stake table + threshold directly

Reply:
Will we always just pass in DA stake table and success_threshold? If so it isn't clear to me why we need a generic `CERT`.

Reply:
I'm rethinking some of my comments here, but I'll leave some thoughts on zulip